### PR TITLE
Simplify the wrapping code for initializer-expression wrapping

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Followup to https://github.com/dotnet/roslyn/pull/39878

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
@sharwell  this is ready to rereview.

**Comment by CyrusNajmabadi** [comment]
@davidwengier

File: src/Features/Core/Portable/Wrapping/AbstractSeparatedListWrapper`2+AbstractSeparatedListCodeComputer`1.cs [hunk 1]
```diff
@@ -1,176 +0,0 @@
...
-using Microsoft.CodeAnalysis.Shared.Extensions;
-using Microsoft.CodeAnalysis.Text;
-using Microsoft.CodeAnalysis.Wrapping.SeparatedSyntaxList;
-using Roslyn.Utilities;
-
-namespace Microsoft.CodeAnalysis.Wrapping
-{
-    internal abstract partial class AbstractSeparatedListWrapper<TListSyntax, TListItemSyntax>
-    {
-        protected abstract class AbstractSeparatedListCodeComputer<TWrapper> : AbstractCodeActionComputer<TWrapper>
```
Comment (CyrusNajmabadi) [inline src/Features/Core/Portable/Wrapping/AbstractSeparatedListWrapper`2+AbstractSeparatedListCodeComputer`1.cs:21]: this code just moved back into AbstractSeparatedListWrapper

File: src/Features/Core/Portable/Wrapping/InitializerExpression/AbstractInitializerExpressionWrapper`2+InitializerExpressionCodeActionComputer.cs [hunk 2]
```diff
@@ -1,210 +0,0 @@
...
-            {
-                using var _ = ArrayBuilder<WrappingGroup>.GetInstance(out var result);
-                await AddWrappingGroupsAsync(result).ConfigureAwait(false);
-                return result.ToImmutableAndClear();
-            }
-
-            protected sealed override ImmutableArray<Edit> GetWrapEachEdits(
-                WrappingStyle wrappingStyle, SyntaxTrivia indentationTrivia)
-            {
-                using var _ = ArrayBuilder<Edit>.GetInstance(out var result);
```
Comment (CyrusNajmabadi) [inline src/Features/Core/Portable/Wrapping/InitializerExpression/AbstractInitializerExpressionWrapper`2+InitializerExpressionCodeActionComputer.cs:59]: all this logic was basically teh same as normal separated wrapping, just with some tweaks about how hte start/end brace are handled.  so this merged back into that, and that code was updated to be hte same as before, just with specialization around start/end brace.

**Review by davidwengier — APPROVED** [review]
Assuming the absence of tests here means no behaviour change

**Comment by CyrusNajmabadi** [comment]
> Assuming the absence of tests here means no behaviour change

Yup. It was just a simplification refactoring

**Pull request merged**